### PR TITLE
chore(deps): Included dependency review

### PR DESCRIPTION
**Pull request by naveensrinivasan**
> Dependency Review GitHub Action in your repository to enforce dependency reviews on your pull requests.
> The action scans for vulnerable versions of dependencies introduced by package version changes in pull requests,
> and warns you about the associated security vulnerabilities.
> This gives you better visibility of what's changing in a pull request,
> and helps prevent vulnerabilities being added to your repository.

https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement
Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

File: .github/workflows/depsreview.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@3f943b86c9a289f4e632c632695e2e0898d9d67d
```
Comment (cpanato) [inline .github/workflows/depsreview.yml:28]: thanks
please add a comment for each action to say in which tag it is in use

Comment (naveensrinivasan) [reply .github/workflows/depsreview.yml:28]: They don’t get updated when Dependabot updates it. 

So it becomes obsolete. 

Let me know.

**Review by cpanato — CHANGES_REQUESTED** [review]
small request

**Comment by naveensrinivasan** [comment]
@cpanato A friendly ping.

File: .github/workflows/depsreview.yml (hunk 1 above)
Comment (cpanato) [reply .github/workflows/depsreview.yml:28]: can you add the tag as a comment like you did for the other PR? just to keep consistent thanks

**Comment by naveensrinivasan** [comment]
> @naveensrinivasan can you add the tag as a comment like you did for the other PR? just to keep consistent thanks

@cpanato  Thanks took care of it!

**Review by cpanato — APPROVED** [review]

**Pull request merged**